### PR TITLE
[tests]: Specify versions of pip packages

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**What I did**
Updated readme file for tests

**Why I did it**
Because the instructions there doesn't work anymore. zipp and pytest packages abandoned python 2 support.

**How I verified it**
Try it on clean system

**Details if related**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ SWSS, Redis, and all the other required components run inside a virtual switch D
     ```
     sudo modprobe team
     sudo apt install net-tools ethtool
-    sudo pip install docker pytest flaky redis
+    sudo pip install docker zipp==2.2.1 pytest==4.6.9 pytest flaky redis
```
Comment (daall) [inline tests/README.md]: `pytest` is listed twice here.

Comment (pavel-shirshov) [reply tests/README.md]: should be fixed now. Thanks for the report!

**Comment by daall** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by qiluo-msft** [comment]
zipp seems broken.
```
$ sudo pip install docker zipp==2.2.1 pytest==4.6.9 flaky redis
Requirement already satisfied: docker in /usr/local/lib/python2.7/dist-packages
Collecting zipp==2.2.1
  Could not find a version that satisfies the requirement zipp==2.2.1 (from versions: 0.1.0, 0.2.0, 0.2.1, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.4.0, 0.5.0, 0.5.1, 0.5.2, 0.6.0, 1.0.0, 1.1.0, 1.1.1, 1.2.0)
No matching distribution found for zipp==2.2.1
```

**Comment by pavel-shirshov** [comment]
Ihttps://pypi.org/project/zipp/#history It was released 2 month ago and still available from pypi.
Can you please try with debug?
What is your pip version?

**Pull request merged**